Microsoft.Toolkit/Scalar.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Toolkit
+{
+    /// <summary>
+    /// Class containing some constants
+    /// represented as Floating point numbers.
+    /// </summary>
+    public static class Scalar
```

Comment:
I'm not sure I like the idea of adding a class such as this to the base package, considering it'll then be visible to literally anyone using the toolkit. The BCL already exposes some of these constants, and for others there are already APIs to facilitate doing conversions. Others can just be typed manually, which is also shorter than typing eg. `Scalar.PiByThree` anyway (also with the extra `using` statement) 🤔

Reply:
I made the class `internal`